docs/assets.md

```diff
@@ -0,0 +1,47 @@
+## Link sprocket assets
```

Comment:
Let's fix all these heading levels. Should start with `#` and step into `##`, etc.

Reply:
Yepp, will do - just moving them into files now and then format 👍